### PR TITLE
Traceview find: background color and prev/next

### LINKED ISSUE #46628: Traceview find: background color and prev/next
Two changes;
- Update color of found span rows (making the rows easier to see),
- Add previous and next buttons to the trace view find (making it easier to navigate large traces).

More details in ['Improve Find'](https://docs.google.com/document/d/1Gke6jGTQ-gM3rY9zvVEc20c4hrs1jm6Z_t1GfI__SlY/edit#) of the large trace viewing document.

### PR DESCRIPTION
**Pull request by joey-grafana**
**What this PR does / why we need it**:
Two changes;
- Update color of found span rows (making the rows easier to see),
- Add previous and next buttons to the trace view find (making it easier to navigate large traces).

More details in ['Improve Find'](https://docs.google.com/document/d/1Gke6jGTQ-gM3rY9zvVEc20c4hrs1jm6Z_t1GfI__SlY/edit#) of the large trace viewing document.

**Which issue(s) this PR fixes**:

Fixes https://github.com/grafana/grafana/issues/46628

**Special notes for your reviewer**:
Color before (second span hovered)
Color before
Color after (first span hovered)
Color after
Sticky search (as we don't want the user to press next/prev and just jump to a span)
Sticky search

### CODE REVIEW

File: packages/jaeger-ui-components/src/TracePageHeader/TracePageSearchBar.tsx [hunk 1]
```diff
@@ -24,11 +23,24 @@ import UiFindInput from '../common/UiFindInput';
...
   return {
     TracePageSearchBar: css`
       label: TracePageSearchBar;
+      float: right;
+      position: sticky;
+      top: 8px;
+      right: 0;
+      z-index: 5;
+      background: ${theme.colors.background.primary};
+      margin-top: 8px;
```
Comment (connorlindsey) [inline packages/jaeger-ui-components/src/TracePageHeader/TracePageSearchBar.tsx:38]: Kind of hacky, but I wanted to add some margin between search bar and the container - so I added padding + margin to give spacing. Also added box-shadow/border-radius to distinguish the search bar a bit more. 

Would be cool in the future to detect when it's scrolled and apply the box-shadow then.

**Review by connorlindsey — APPROVED** [review]
Made a few changes to the styling and removed the clear button to use the Input's clear instead. Let me know what you think

File: packages/jaeger-ui-components/src/TracePageHeader/TracePageSearchBar.tsx [hunk 2]
```diff
@@ -24,11 +23,24 @@ import UiFindInput from '../common/UiFindInput';
...
-export const getStyles = () => {
+export const getStyles = (theme: GrafanaTheme2) => {
   return {
     TracePageSearchBar: css`
       label: TracePageSearchBar;
+      float: right;
+      position: sticky;
+      top: 8px;
+      right: 0;
+      z-index: 5;
```
Comment (ifrost) [inline packages/jaeger-ui-components/src/TracePageHeader/TracePageSearchBar.tsx:36]: Would it make sense to use `${theme.zIndex....}` here?

File: packages/jaeger-ui-components/src/TracePageHeader/TracePageSearchBar.tsx [hunk 3]
```diff
@@ -61,72 +73,62 @@ export const getStyles = () => {
...
+    </span>
+  ) : null;
 
   const btnClass = cx(styles.TracePageSearchBarBtn, { [styles.TracePageSearchBarBtnDisabled]: !searchValue });
   const uiFindInputInputProps = {
     'data-test': markers.IN_TRACE_SEARCH,
     className: cx(styles.TracePageSearchBarBar, ubFlexAuto),
     name: 'search',
-    suffix: count,
+    suffix: suffix,
```
Comment (ifrost) [inline packages/jaeger-ui-components/src/TracePageHeader/TracePageSearchBar.tsx:97]:
```suggestion
    suffix,
```

File: public/app/features/explore/TraceView/TraceViewContainer.tsx [hunk 4]
```diff
@@ -13,18 +16,153 @@ interface Props {
...
+          childrenToggle={childrenToggle}
+          childrenHiddenIDs={childrenHiddenIDs}
+          queryResponse={queryResponse}
+        />
+      </Collapse>
+    </>
   );
 }
+
+export function transformDataFrames(frame?: DataFrame): Trace | null {
```
Comment (ifrost) [inline public/app/features/explore/TraceView/TraceViewContainer.tsx:120]: Looks like a utility function. Could we move it out from to `utils`?

**Review by ifrost — CHANGES_REQUESTED** [review]
Looks good, just left a few minor comments.

**Review by ifrost — APPROVED** [review]
🚀

**Pull request merged**